data/navneet17360.json

```
@@ -0,0 +1,16 @@
+{
+	"name": "Navneet Anand",
+	"type": "community",
+	"bio": "Front-End Developer",
+	"links": [{
+			"name": "Follow me on Instagram",
+			"url": "https://www.instagram.com/navneet_anand_17",
+			"icon": "FaInstagram"
+		},
+		{
+			"name": "Follow me on Github",
+			"url":"https://github.com/navneet17360",
+			"icon:"FaGithub"
```

needs the following corrections:
```suggestion
			"icon": "FaGithub",
```